src/SW_SoilWater.c

```diff
@@ -304,7 +304,7 @@ static double itp_FXW_for_phi(double theta, double *swrcp, LOG_INFO* LogInfo) {
 
 
 	#ifdef SWDEBUG
-	if (debug) {
+	if (debug && !LogInfo->stopRun) {

```

Comment:
please remove the ` && !LogInfo->stopRun` from `if (debug && !LogInfo->stopRun) {` in "SWDEBUG" section of `itp_FXW_for_phi()` -- that debug output was intended to be printed on error (see the else branch of `!missing(phi)`)